modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/defragmentation/LinkMap.java

```diff
@@ -45,12 +41,12 @@
     public static final int META_PAGE_IDX = 2;
 
     /** */
-    public static final IOVersions<? extends BPlusLeafIO<?>> LEAF_IO_VERSIONS = new IOVersions<>(
+    public static final IOVersions<LinkMappingLeafIO> LEAF_IO_VERSIONS = new IOVersions<>(
```

Comment:
This was deliberate, IO classes are private. If you have a better option then go ahead